apps/accounts/forms.py

```diff
@@ -332,14 +328,6 @@ def clean_email(self):
         else:
             return email.rstrip().lstrip().lower()
 
-    def clean_username(self):
-        username = self.cleaned_data.get('username')
-        username = username.rstrip().lstrip().lower()
-
-        if User.objects.filter(username=username).count() > 0:
-            raise forms.ValidationError(_('This username is already taken.'))
```

Comment:
Is this logic to warn of duplicates still taken care of?

Reply:
Email doesn't allow duplicates.  Its not in the DB. but the logic is in the app.  (You could break it with the admin if not careful).